README.md

```diff
@@ -357,14 +366,17 @@ When adding a new app, add it to both the [release-please-config.json](./release
 
 ### Troubleshoting release-please
 
-If you were expecting a new release PR to be created, but nothing happened, there's probably one of the older release PR's in pending state.
-
-First make sure there's no open release PR already; if there is, the work is now included on this one (this is the normal scenario).
+If you were expecting a new release PR to be created or old updated, but nothing happened, there's probably one of the older release PR's in pending state or action didn't run.
```

Comment:
How about "If you were expecting a new release PR to be created or old updated" → "If you were expecting a new release PR to be created or an old one to be updated"?